### DISCUSSION TITLE
Use of templateUrl vs template for content projection

### DISCUSSION THREAD

**Discussion by paul-request** [Legal-and-General/canopy#335] [General]
I've seen a few instances recently where we use a component to project content into a certain area of another component. E.g.

```
<lg-omponent>
  <lg-content>
    {{content}}
  </lg-content>
</lg-component>
```

The `lg-content` component does nothing other than project the content, so it's template is simply `<ng-content></ng-content>`. In this instance it seems to me that we may as well put that directly into the component `template` definition, rather than adding to a separate file via `templateUrl`.

Would like to know if anyone has an issue with this, or has a different opinion?

**Comment by owensgit** [comment]
I would be fine with this 👍
